src/ApiClientProvider.php

```diff
@@ -69,6 +71,8 @@ private function createClient(array $options = []) : BEditaClient
         $this->apiClient = new BEditaClient(Configure::read('API.apiBaseUrl'), Configure::read('API.apiKey'));
         if (!empty($options['Log'])) {
```

Comment:
alternatively:

```php
$logOptions = !empty($options['log']) ? : $options['Log'] : Configure::read('API.log'):
if (!empty($logOptions)) {
...
}
```
maybe not elegant, but probably already tested /easier to test :wink: